test/extended/oauth/requestheaders.go

```diff
@@ -504,32 +508,3 @@ func getTokenFromResponse(resp *http.Response) string {
 
 	return ""
 }
-
-func waitForAuthenticationProgressing(oc *exutil.CLI, expectedProgressing configv1.ConditionStatus) {
-	err := wait.PollImmediate(time.Second, 10*time.Minute, func() (bool, error) {
-		authn, err := oc.AdminConfigClient().ConfigV1().ClusterOperators().Get(context.Background(), "authentication", metav1.GetOptions{})
-		if err != nil {
-			e2e.Logf("Error getting authentication operator: %v", err)
-			return false, err
-		}
-
-		progressing := clusteroperatorhelpers.FindStatusCondition(authn.Status.Conditions, configv1.OperatorProgressing)
-		if progressing == nil || progressing.Status != expectedProgressing {
-			e2e.Logf("Waiting for progressing condition to be %q: %s", expectedProgressing, spew.Sdump(authn.Status.Conditions))
-			return false, nil
-		}
-
-		if expectedProgressing == configv1.ConditionFalse {
```

Comment:
I don't see this reflected in your changes:

```mermaid
flowchart LR
progressing["progressing: false"]
available["available: true"]
degraded["degraded: false"]
buggy["mermaid is buggy"]

progressing -->|AND| available
progressing -->|AND| degraded
```

Reply:
I don't think the degraded check was doing what you thought it was doing.  You need to be sure the level is achieved, but I wouldn't expect a degraded on a two minute momentum to have a meaningful impact for you